### PR TITLE
[Dashboard] Remove AddressCopyButton

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR focuses on removing the `AddressCopyButton` component from various files and replacing it with the `CopyTextButton` and `CopyAddressButton` components to optimize bundle size and improve code consistency.

### Detailed summary
- Deleted `AddressCopyButton.tsx`.
- Removed all imports of `AddressCopyButton` from multiple files.
- Replaced instances of `AddressCopyButton` with `CopyTextButton` or `CopyAddressButton` in:
  - `webhooks-table.tsx`
  - `token-id.tsx`
  - `transaction-timeline.tsx`
  - `contract-subscriptions-table.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5077?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#5077** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5077?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#5035** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5035?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5076** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5076?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/components/engine/configuration/webhooks-table.tsx [hunk 1]
```diff
@@ -58,10 +58,11 @@ const columns = [
     header: "Secret",
     cell: (cell) => {
       return (
-        <AddressCopyButton
-          address={cell.getValue() || ""}
-          title="secret"
-          size="xs"
+        <CopyTextButton
```
Comment (MananTank) [inline apps/dashboard/src/components/engine/configuration/webhooks-table.tsx:61]: `AddressCopyButton` trims the text shown on the button - which you have to do manually on `CopyTextButton` 

production: 



This PR: 




please make sure the `textToShow` is trimmed properly on all changes done in this PR

**Review by MananTank — CHANGES_REQUESTED** [review]
please address comments

File: apps/dashboard/src/components/engine/configuration/webhooks-table.tsx (hunk 1 above)
Comment (kien-ngo) [reply apps/dashboard/src/components/engine/configuration/webhooks-table.tsx:61]: oh, ty ser

File: apps/dashboard/src/components/engine/overview/transaction-timeline.tsx [hunk 2]
```diff
@@ -242,10 +242,9 @@ const CancelTransactionButton = ({
               </FormControl>
               <FormControl>
                 <FormLabel>To</FormLabel>
-                <AddressCopyButton
+                <CopyAddressButton
```
Comment (MananTank) [inline apps/dashboard/src/components/engine/overview/transaction-timeline.tsx:245]: use the new `WalletAddress` component instead that shows social profiles

File: apps/dashboard/src/components/engine/contract-subscription/contract-subscriptions-table.tsx [hunk 3]
```diff
@@ -93,9 +93,9 @@ export const ContractSubscriptionTable: React.FC<
         const explorer = chain?.explorers?.[0];
         if (!explorer) {
           return (
-            <AddressCopyButton
+            <CopyAddressButton
```
Comment (MananTank) [inline apps/dashboard/src/components/engine/contract-subscription/contract-subscriptions-table.tsx:96]: use the new `WalletAddress` component instead that shows social profiles

Comment (kien-ngo) [reply apps/dashboard/src/components/engine/contract-subscription/contract-subscriptions-table.tsx:96]: @MananTank this is for showing contract address btw, do we want to use the "social" feature on them?

I will update the code for u to review anyway

Comment (MananTank) [reply apps/dashboard/src/components/engine/contract-subscription/contract-subscriptions-table.tsx:96]: Ahh may bad. Thought this was wallet addess

**Review by MananTank — APPROVED** [review]

**Pull request merged**